### ISSUE TITLE
Search for reports on a map by police authority

### ISSUE THREAD

**Issue by LouiseMySociety** [mysociety/collideoscope#67]
Add ability to find police authorities to the 'Show reports on a map' drop down box on the reports page.

**Comment by LouiseMySociety** [comment]
![Screen Shot 2018-09-07 at 16.35.31.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5aec706e465939001342d64d/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b772a6454f46b7191a379e90c2854415c4fbe5d4a1f06a3a3dc78e2a436628f5459eb83645a27906e19aebed71db801bb73884b87e4ee9850d2aa6c7bb3d4d216744b470a83ed5c32aa984ecb5ef548e5485da61a3f6bf43b.png)

**Comment by LouiseMySociety** [comment]
Agreed to not to implement at the moment and ensure there is a requirement for it through further user testing. 

- [ ] @zarino to remove the 'or police force' text from the label.

**Comment by dracos** [comment]
"@zarino to remove the 'or police force' text from the label." already covered in #61

**Comment by zarino** [comment]
So, interestingly, one of the first bits of feedback I had, after showing the updated site to my original user research subjects, was "It doesn’t let me filter by police area or parliamentary constituency".

It feels like adding this would make the `/reports` page _much_ more useful, since it would enable campaigners to tailor their data to specific PCCs or MPs.
